section4/reflection.md

```diff
@@ -2,20 +2,31 @@
+  It helped my work feel more focused and efficient. During the times I set aside to put in work, I was more deliberate with making progress.
 
-1. What is an Object, and how is it different from an Array in Javascript?
+2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
-1. For each set of data, would an array or object be better to store it? Explain your choice.
+  I thought I was more focused! I would like to try using a mechanical timer, so I get the added pressure of the ticking sound counting down the allotted time.
 
-  * List of all of the students in class
-  * List of states and their capitals
-  * List of things to pack for vacation
-  * Names of all the Instagram accounts I follow
-  * List of student names and their cohort
-  * Ingredients and amount of each ingredient to bake a cake
-  * All my favorite restaurants
+3. What is an Object, and how is it different from an Array in Javascript?
 
-1. In this section, we talked about an `transaction item object`. It has keys that represent different properties an bank transaction would have, and values that store the data. In a banking app, this data would eventually be displayed in the browser for the user. Think of another Object in a web application that you use frequently. What is it? What keys might be on it? What are example values? 
+  An object is something that has individual properties and can have methods as well. In addition, objects contain arrays of properties. An array is an ordered list of values.
 
-1. What questions do you still have about classes and/or Objects?
+4. For each set of data, would an array or object be better to store it? Explain your choice.
+
+  * List of all of the students in class _array_
+  * List of states and their capitals _object_
+  * List of things to pack for vacation _array_
+  * Names of all the Instagram accounts I follow _array_
+  * List of student names and their cohort _object_
+  * Ingredients and amount of each ingredient to bake a cake _object_
+  * All my favorite restaurants _array_
+
+5. In this section, we talked about an `transaction item object`. It has keys that represent different properties an bank transaction would have, and values that store the data. In a banking app, this data would eventually be displayed in the browser for the user. Think of another Object in a web application that you use frequently. What is it? What keys might be on it? What are example values?
+
+  Example: Facebook
+  User Profile: keys might be username, emailAddress, while values might look like "ericli1996" or "ericli@knights.ucf.edu"
+
+6. What questions do you still have about classes and/or Objects?
+
+  I would like to better grasp the syntax for different things I can do with an Object Method.
```

Comment:
Awesome, you will have so much time and practice as you progress at Turing, however, check out these resources I found by researching "practicing objects javascript"
1. https://www.codeanalogies.com/objects-arrays-practice
2. https://github.com/LearnTeachCode/pair-partners/issues/42
3. https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object_building_practice
4. https://www.w3resource.com/javascript-exercises/javascript-object-exercises.php
5. https://www.teaching-materials.org/javascript/exercises/objects